src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.WriterService;
+import java.io.FileWriter;
+import java.util.List;
+
+public class WriterServiceImpl implements WriterService {
```

Comment:
The class WriterServiceImpl should not start with an empty line. Please remove the extra line.